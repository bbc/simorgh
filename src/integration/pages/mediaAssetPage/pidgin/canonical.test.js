/**
 * @service pidgin
 * @pathname /pidgin/23248703
 */

import runCanonicalTests from '../canonicalTests';

describe('Canonical', () => {
  describe(pageType, () => {
    runCanonicalTests(service, pageType);
  });
});
