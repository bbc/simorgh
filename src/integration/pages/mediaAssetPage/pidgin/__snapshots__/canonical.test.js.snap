// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical Media Asset Page A11y I can see the skip to content link 1`] = `"Waka go wetin de inside"`;

exports[`Canonical Media Asset Page Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=70&amp;p=pidgin.media_asset.23248703.page&amp;x2=[responsive]&amp;x3=[news-pidgin]&amp;x4=[pcm]&amp;x7=[article-media-asset]&amp;x8=[simorgh-nojs]&amp;x9=[Simorgh%3A%2520Media%2520Pod%2520Build%2520First%2520CPS%2520Media%2520Asset%2520Page%2520in%2520Simorgh%2520%26%2520%3C%2520%3E%2520-%2520BBC%2520News%2520Pidgin]&amp;x11=[2019-09-13T15%3A31%3A44.000Z]&amp;x12=[2019-11-20T12%3A24%3A25.000Z]&amp;x16=[Inspire%20me~Give%20me%20perspective~Keep%20me%20on%20trend]&amp;x17=[Opinion]\\"/>"`;

exports[`Canonical Media Asset Page Footer Anchors should match text and url 1`] = `
Object {
  "text": "BBC News, Pidgin",
  "url": "/pidgin",
}
`;

exports[`Canonical Media Asset Page Footer Anchors should match text and url 2`] = `
Object {
  "text": "Why you fit trust BBC News",
  "url": "https://www.bbc.com/pidgin/institutional-48528766",
}
`;

exports[`Canonical Media Asset Page Footer Anchors should match text and url 3`] = `
Object {
  "text": "How dem dey take use am",
  "url": "https://www.bbc.co.uk/usingthebbc/terms/",
}
`;

exports[`Canonical Media Asset Page Footer Anchors should match text and url 4`] = `
Object {
  "text": "As e concern BBC",
  "url": "https://www.bbc.co.uk/aboutthebbc",
}
`;

exports[`Canonical Media Asset Page Footer Anchors should match text and url 5`] = `
Object {
  "text": "Privacy Policy",
  "url": "https://www.bbc.co.uk/usingthebbc/privacy/",
}
`;

exports[`Canonical Media Asset Page Footer Anchors should match text and url 6`] = `
Object {
  "text": "Cookies",
  "url": "https://www.bbc.co.uk/usingthebbc/cookies/",
}
`;

exports[`Canonical Media Asset Page Footer Anchors should match text and url 7`] = `
Object {
  "text": "Call BBC",
  "url": "https://www.bbc.co.uk/pidgin/send/u50853577",
}
`;

exports[`Canonical Media Asset Page Footer Anchors should match text and url 8`] = `
Object {
  "text": "AdChoices / Do Not Sell My Info",
  "url": "https://www.bbc.com/usingthebbc/cookies/how-does-the-bbc-use-cookies-for-advertising/",
}
`;

exports[`Canonical Media Asset Page Footer Anchors should match text and url 9`] = `
Object {
  "text": "De way wey we de take go external link.",
  "url": "https://www.bbc.co.uk/editorialguidelines/guidance/feeds-and-links",
}
`;

exports[`Canonical Media Asset Page Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/pidgin"`;

exports[`Canonical Media Asset Page Footer I can see the footer copyright and external linking text 1`] = `"Â© 2023 BBC. De external site no concern BBC. De way wey we de take go external link."`;

exports[`Canonical Media Asset Page Header I can see the branding 1`] = `"BBC News, Pidgin"`;

exports[`Canonical Media Asset Page Header Navigation link should match text and url 1`] = `
Object {
  "text": "Home",
  "url": "/pidgin",
}
`;

exports[`Canonical Media Asset Page Header Navigation link should match text and url 2`] = `
Object {
  "text": "Nigeria",
  "url": "/pidgin/topics/c2dwqd1zr92t",
}
`;

exports[`Canonical Media Asset Page Header Navigation link should match text and url 3`] = `
Object {
  "text": "Africa",
  "url": "/pidgin/topics/c404v061z85t",
}
`;

exports[`Canonical Media Asset Page Header Navigation link should match text and url 4`] = `
Object {
  "text": "World",
  "url": "/pidgin/topics/c0823e52dd0t",
}
`;

exports[`Canonical Media Asset Page Header Navigation link should match text and url 5`] = `
Object {
  "text": "Video",
  "url": "/pidgin/topics/c3l19z3k1ert",
}
`;

exports[`Canonical Media Asset Page Header Navigation link should match text and url 6`] = `
Object {
  "text": "Sport",
  "url": "/pidgin/topics/cjgn7gv77vrt",
}
`;

exports[`Canonical Media Asset Page Header Navigation link should match text and url 7`] = `
Object {
  "text": "Entertainment",
  "url": "/pidgin/topics/cqywjyzk2vyt",
}
`;

exports[`Canonical Media Asset Page Header Navigation link should match text and url 8`] = `
Object {
  "text": "Most popular",
  "url": "/pidgin/popular/read",
}
`;

exports[`Canonical Media Asset Page I can see a bulleted list item 1`] = `"<Media Asset Page>"`;

exports[`Canonical Media Asset Page I can see a bulleted list item with link: <Media Asset Page> 1`] = `"/afrique/23248423"`;

exports[`Canonical Media Asset Page Main heading should match text 1`] = `"Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh & < >"`;

exports[`Canonical Media Asset Page Media Player a11y assistive technology can read the media player title 1`] = `"Media player"`;

exports[`Canonical Media Asset Page Media Player iframe with valid URL should be rendered 1`] = `"https://www.test.bbc.com/ws/av-embeds/cps/pidgin/23248703/p01kx42v/pcm"`;

exports[`Canonical Media Asset Page Related Content should match text and url 1`] = `
Object {
  "text": "Police Arrest 3 ontop Anambra Church Attack",
  "url": "/pidgin/tori-23145776",
}
`;

exports[`Canonical Media Asset Page Related Content should match text and url 2`] = `
Object {
  "text": "Lagos 'Gay' men Appear for Court, say Dem no dey Guilty",
  "url": "/pidgin/tori-23143754",
}
`;

exports[`Canonical Media Asset Page Related Content should match text and url 3`] = `
Object {
  "text": "Image gallery, Kenya Elections Pictures: 8 August 2017",
  "url": "/pidgin/23146654",
}
`;

exports[`Canonical Media Asset Page Related Content should match text and url 4`] = `
Object {
  "text": "Make you listen, BBC News Pidgin One Minute News, Duration 0,15",
  "url": "/pidgin/media-23147054",
}
`;

exports[`Canonical Media Asset Page SEO Apple Touch Icon should match attributes 1`] = `
Object {
  "sizes": null,
  "url": "http://localhost:7080/pidgin/images/icons/icon-192x192.png",
}
`;

exports[`Canonical Media Asset Page SEO Apple Touch Icon should match attributes 2`] = `
Object {
  "sizes": "72x72",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-72x72.png",
}
`;

exports[`Canonical Media Asset Page SEO Apple Touch Icon should match attributes 3`] = `
Object {
  "sizes": "96x96",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-96x96.png",
}
`;

exports[`Canonical Media Asset Page SEO Apple Touch Icon should match attributes 4`] = `
Object {
  "sizes": "128x128",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-128x128.png",
}
`;

exports[`Canonical Media Asset Page SEO Apple Touch Icon should match attributes 5`] = `
Object {
  "sizes": "144x144",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-144x144.png",
}
`;

exports[`Canonical Media Asset Page SEO Apple Touch Icon should match attributes 6`] = `
Object {
  "sizes": "152x152",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-152x152.png",
}
`;

exports[`Canonical Media Asset Page SEO Apple Touch Icon should match attributes 7`] = `
Object {
  "sizes": "180x180",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-180x180.png",
}
`;

exports[`Canonical Media Asset Page SEO Apple Touch Icon should match attributes 8`] = `
Object {
  "sizes": "192x192",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-192x192.png",
}
`;

exports[`Canonical Media Asset Page SEO Apple Touch Icon should match attributes 9`] = `
Object {
  "sizes": "384x384",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-384x384.png",
}
`;

exports[`Canonical Media Asset Page SEO Apple Touch Icon should match attributes 10`] = `
Object {
  "sizes": "512x512",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/pidgin/images/icons/icon-512x512.png",
}
`;

exports[`Canonical Media Asset Page SEO Canonical link 1`] = `"http://localhost:7080/pidgin/23248703"`;

exports[`Canonical Media Asset Page SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical Media Asset Page SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical Media Asset Page SEO Heading level 1 1`] = `"Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh & < >"`;

exports[`Canonical Media Asset Page SEO Lang attribute 1`] = `"pcm"`;

exports[`Canonical Media Asset Page SEO Linked data should match text 1`] = `
Object {
  "@context": "http://schema.org",
  "@graph": Array [
    Object {
      "@type": "Article",
      "author": Object {
        "@type": "NewsMediaOrganization",
        "logo": Object {
          "@type": "ImageObject",
          "height": 576,
          "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
          "width": 1024,
        },
        "name": "BBC News Pidgin",
        "noBylinesPolicy": "https://www.bbc.com/pidgin/institutional-48528766#authorexpertise",
      },
      "dateModified": "2019-11-20T12:24:25.000Z",
      "datePublished": "2019-09-13T15:31:44.000Z",
      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh & < >",
      "image": Object {
        "@type": "ImageObject",
        "height": 576,
        "url": "https://ichef.bbci.co.uk/news/1024/branded_pidgin/6FC4/test/_63721682_p01kx435.jpg",
        "width": 1024,
      },
      "inLanguage": Object {
        "@type": "Language",
        "alternateName": "pcm",
        "name": "Nigerian Pidgin",
      },
      "mainEntityOfPage": Object {
        "@id": "http://localhost:7080/pidgin/23248703",
        "@type": "WebPage",
        "name": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh & < >",
      },
      "publisher": Object {
        "@type": "NewsMediaOrganization",
        "logo": Object {
          "@type": "ImageObject",
          "height": 576,
          "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
          "width": 1024,
        },
        "name": "BBC News Pidgin",
        "publishingPrinciples": "https://www.bbc.com/pidgin/institutional-48528766",
      },
      "thumbnailUrl": "https://ichef.bbci.co.uk/news/1024/branded_pidgin/6FC4/test/_63721682_p01kx435.jpg",
      "url": "http://localhost:7080/pidgin/23248703",
    },
  ],
}
`;

exports[`Canonical Media Asset Page SEO Linked data should match text 2`] = `
Object {
  "@context": "http://schema.org",
  "@type": "VideoObject",
  "description": "short",
  "duration": "PT5S",
  "embedURL": "https://www.test.bbc.com/ws/av-embeds/cps/pidgin/23248703/p01kx42v/pcm",
  "name": "alt-text world service clip",
  "thumbnailUrl": "https://ichef.test.bbci.co.uk/images/ic/1024x576/p01kx435.jpg",
  "uploadDate": "2018-12-20T11:06:51.000Z",
}
`;

exports[`Canonical Media Asset Page SEO OG description 1`] = `"This week, CPS media assets were visible in Simorgh for the first time. The pages were built out in a matter of weeks using existing Psammead Components."`;

exports[`Canonical Media Asset Page SEO OG image 1`] = `"https://ichef.bbci.co.uk/news/1024/branded_pidgin/6FC4/test/_63721682_p01kx435.jpg"`;

exports[`Canonical Media Asset Page SEO OG image alt 1`] = `"connectionAltText"`;

exports[`Canonical Media Asset Page SEO OG locale 1`] = `"pcm"`;

exports[`Canonical Media Asset Page SEO OG site name 1`] = `"BBC News Pidgin"`;

exports[`Canonical Media Asset Page SEO OG title 1`] = `"First CPS Media Asset Pages Appear on Heloo - BBC News Pidgin"`;

exports[`Canonical Media Asset Page SEO OG type 1`] = `"article"`;

exports[`Canonical Media Asset Page SEO OG url 1`] = `"http://localhost:7080/pidgin/23248703"`;

exports[`Canonical Media Asset Page SEO Page title 1`] = `"Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh & < > - BBC News Pidgin"`;

exports[`Canonical Media Asset Page SEO Robots meta tag 1`] = `"noodp, noydir, max-image-preview:large"`;

exports[`Canonical Media Asset Page SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical Media Asset Page SEO Twitter creator 1`] = `"@BBCNews"`;

exports[`Canonical Media Asset Page SEO Twitter description 1`] = `"This week, CPS media assets were visible in Simorgh for the first time. The pages were built out in a matter of weeks using existing Psammead Components."`;

exports[`Canonical Media Asset Page SEO Twitter image alt 1`] = `"connectionAltText"`;

exports[`Canonical Media Asset Page SEO Twitter image src 1`] = `"https://ichef.bbci.co.uk/news/1024/branded_pidgin/6FC4/test/_63721682_p01kx435.jpg"`;

exports[`Canonical Media Asset Page SEO Twitter site 1`] = `"@BBCNews"`;

exports[`Canonical Media Asset Page SEO Twitter title 1`] = `"First CPS Media Asset Pages Appear on Heloo - BBC News Pidgin"`;

exports[`Canonical Media Asset Page Timestamp should match text and date 1`] = `
Object {
  "date": "2019-09-13",
  "text": "13 September 2019",
}
`;
